incubator/hnc/cmd/manager/main.go

```diff
@@ -67,11 +69,17 @@ func main() {
 	flag.BoolVar(&newObjectController, "enable-new-object-controller", false, "Enables new object controller.")
 	flag.BoolVar(&testLog, "enable-test-log", false, "Enables test log.")
 	flag.IntVar(&maxReconciles, "max-reconciles", 1, "Number of concurrent reconciles to perform.")
+	flag.IntVar(&qps, "qps", 50, "The maximum QPS to the API server.")
+	flag.IntVar(&burst, "burst", 75, "Maximum burst for throttle.")
```

Comment:
I think my preference would be just have a single number (qps) and always set "burst" to maybe qps*1.5.

Also, "--qps" doesn't really mean a lot on the command line - it's not self-descriptive. How about `--apiserver-qps-throttle`? That makes it clear what we're protecting with this flag. Internally we can still call it the same thing.

Reply:
Thanks for the suggestion. Please see the updated code.